cmd/sops/main.go

```diff
@@ -552,6 +552,10 @@ func main() {
 			Name:  "rotate, r",
 			Usage: "generate a new data encryption key and reencrypt all values with the new key",
 		},
+		cli.BoolFlag{
+			Name:  "manage, m",
+			Usage: "manage master keys without reencrypting all values with a new data key",
```

Comment:
I think this requires a BIG FAT SECURITY WARNING for the case that you actually want to remove master keys. Or maybe the code should simply fail if a master key is removed and this flag is set.

Reply:
Yes, good point - removing a master key one is probably happening because the key has been compromised or is no longer in use, in which case it shouldn't be trusted anyway. I will update the PR.